### PR TITLE
Adopters, CNCF Banner Sections and Chaoshub Page

### PR DESCRIPTION
**Pull request by mallick370**
*Issue in tablet responsiveness.*
*Few texts are in Lorem Ipsum needs to be changed before release.*
*For mobile and laptops/desktop responsiveness almost done.*
*Updated Litmus Logo and also added required images and icons*

Signed-off-by: Mayukh Mallick <mallick370@gmail.com>

### CODE REVIEW

File: src/components/adopter/adopter.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+const CreatedByColumn: React.FC = () => {
+  const { sm } = useTheme().screens
+  if (sm) {
+    return <div style={{ margin: '0 1rem' }}>
+      <h5 className="AdopterHeading">Created By</h5>
+            <Mayadata />
+    </div>
+  } else {
+    return (
+      <div style={{ margin:'0 0 3rem 0' }}>
```
Comment (arkajyotiMukherjee) [inline src/components/adopter/adopter.tsx:99]: use margin bottom instead

File: src/components/adopter/adopter.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+}
+
+
+const Adopter: React.FC = () => {
+    
+  
+    
+    return (
+    <>
+      <ResponsiveRow>
```
Comment (arkajyotiMukherjee) [inline src/components/adopter/adopter.tsx:164]: the components are maintaining social distancing....so much empty space is not required :)

File: src/components/banner/banner.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.LeftColumn
+{
+    
+    max-width: 791px;
```
Comment (arkajyotiMukherjee) [inline src/components/banner/banner.css:4]: use rems instead of px

File: src/components/adopter/adopter.tsx (hunk 1 above)
Comment (mallick370) [reply src/components/adopter/adopter.tsx:99]: Will surely resolve the issue

File: src/components/adopter/adopter.tsx (hunk 2 above)
Comment (mallick370) [reply src/components/adopter/adopter.tsx:164]: Will surely resolve the issue

File: src/components/banner/banner.css (hunk 3 above)
Comment (mallick370) [reply src/components/banner/banner.css:4]: Okay

File: src/components/adopter/adopter.tsx (hunk 2 above)
Comment (mallick370) [reply src/components/adopter/adopter.tsx:164]: Updated the file with px values to rem values

File: src/components/banner/banner.css (hunk 3 above)
Comment (mallick370) [reply src/components/banner/banner.css]: Updated the file with px values to rem values

File: src/components/adopter/adopter.tsx (hunk 1 above)
Comment (mallick370) [reply src/components/adopter/adopter.tsx]: Done

File: src/components/adopter/adopter.tsx (hunk 2 above)
Comment (mallick370) [reply src/components/adopter/adopter.tsx:164]: Done

**Pull request closed without merging**